app/views/games/show.html.erb

```diff
@@ -14,9 +14,23 @@
   </div>
 </div>
 
+<script src="https://www.gstatic.com/firebasejs/3.7.6/firebase-app.js"></script>
+<script src="https://www.gstatic.com/firebasejs/3.7.6/firebase-database.js"></script>
+<script>
+  // Initialize Firebase
+  var config = {
+    apiKey: '<%= ENV["apiKey"] %>',
```

Comment:
Is there a better way to initialize Firebase? A user could discover the api key simply by looking at the source html in their browser.